src/plugins/plot/axis/YAxis.vue

```diff
@@ -113,18 +119,20 @@ export default {
             return this.singleSeries === true || this.hasSameRangeValue === true;
         },
         yAxisStyle() {
-            const isMainYAxis = this?.id === this.mainYAxisId;
             let style;
             const width = `width: ${this.tickWidth + 20}px`;
-            const border = `border-right: 1px solid`;
-            if (isMainYAxis) {
-                if (this.visibleYAxes > 1) {
-                    style = `${width}; left: ${this.plotWidth - this.tickWidth + 40}px`;
+            const multipleTickOffset = this.multipleLeftAxes ? 20 : 0;
+            const border = this.multipleLeftAxes ? `border-right: 1px solid` : '';
+
+            if (this.position === 'left') {
+                const offset = (this.id - 1);
+                if (offset) {
+                    style = `${width}; ${border}; left: ${this.plotLeftTickWidth - this.tickWidth - offset * (multipleTickOffset + 5)}px`;
```

Comment:
Let's convert this to return a style object and clean up the magic numbers, see above comment on xAxisStyle

Reply:
Done